src/app/shared/components/dropdown/dropdown.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, EventEmitter, Output, Input } from '@angular/core';
+
+@Component({
+  selector: 'ccf-dropdown',
+  templateUrl: './dropdown.component.html',
+  styleUrls: ['./dropdown.component.scss']
+})
+export class DropdownComponent {
+  @Input() label: string;
+  @Input() options: string;
```

Comment:
This should be string[]. You don't need to do string manipulations. in the html template you can do this: `<ccf-dropdown [options]="['1', '2', '3']" ...></ccf-dropdown>`